.github/environments/disabled/requirements.txt

```diff
@@ -0,0 +1,3 @@
+tutor[full]<18
```

Comment:
Could it be better to have just 1 requirements file shared across all the cases? The only difference between the requirements is that in a couple of them, S3 plugin is installed and on the other it's absent. We can install all we need from a single requirements file and decide what plugins to include or not in every case through the `PLUGINS` variable of the `config.yml`